src/lib/parser/generateCodeSamples.ts

```
@@ -3,6 +3,7 @@
 import { availableLanguages, useTheme } from '../../composables/useTheme'
 import { buildRequest } from '../codeSamples/buildRequest'
 import { generateCodeSample } from '../codeSamples/generateCodeSample'
+import { getPropertyExample } from '../examples/getPropertyExample'
```

ESLint

[error] 'getPropertyExample' is defined but never used.

(unused-imports/no-unused-imports)